src/pages/verify/e-ids/danish-mitid.mdx

```diff
@@ -19,21 +19,28 @@ import DanishMitIDAddressJwtSnippet from '../../../snippets/jwt-payloads/danish-
 
 ### MitID for citizens (with address lookup)
 
-Address lookups is available as an add-on for MitID.
+[Address lookups](/verify/e-ids/danish-mitid/#collecting-user-addresses) is available as an add-on for MitID.
 
 <DanishMitIDAddressJwtSnippet />
 
 ## Test users
 
 You create personal MitID test users at [https://pp.mitid.dk/test-tool/frontend](https://pp.mitid.dk/test-tool/frontend):
 
-1. Just use the autofill button and then change the details to you liking
+1. Just use the autofill button and then change the details to you liking.
 2. If you haven't done so already, install the test app from [https://pp.mitid.dk/mitid-app/index.html](https://pp.mitid.dk/mitid-app/index.html)
 _**Note:** On iOS, you can install both the test and production versions of the MitID app on the same device. On Android, the two versions cannot coexist, so you'll need a separate device for testing._
 3. You can test your new users by using the "Flows" menu at the top of the screen.
 
 ![MitID test tool](./images/mitid-testtool.png)
 
+### Adding address data to a test user
+
+To test [address lookups](/verify/e-ids/danish-mitid/#collecting-user-addresses), you can add fictitious address data to your test user via the Dashboard.
+Navigate to **Identity Providers** > **DK MitID** > **Test Users**, and register your test user with a new address.
+![Adding test user addresses](./images/mitid-test-user-addresses.png)
+_Note: To access this feature, you must first fill in the MitID application form at **Identity Providers** > **DK MitID**._
```

Comment:
Maybe put this in a highlight box to make it more noticeable?

Reply:
Agree! thanks :)